src/SFA.DAS.ApplyService.Web.UnitTests/Services/OverallOutcomeServiceTests.cs

```diff
@@ -319,8 +322,38 @@ public async Task BuildApplilcationSummaryViewModel_builds_expected_viewModel()
 
             var returnedModel = _service.BuildApplicationSummaryViewModel(application, emailAddress);
             expectedModel.Should().BeEquivalentTo(returnedModel);
-        } 
-       private List<AssessorSequence> SetUpAsessorSequences()
+        }
+
+        [Test]
+        public async Task moderator_pass_with_failed_moderator_questions_returns_model_with_no_sequences_and_external_comments ()
+        {
+            var submittedApp = new Apply
+            {
+                ApplicationStatus = ApplicationStatus.Unsuccessful,
+                ApplicationId = _applicationId,
+                GatewayReviewStatus = GatewayReviewStatus.Pass,
+                ModerationStatus = ModerationStatus.Pass,
+                ApplyData = new ApplyData
+                {
+                    ApplyDetails = new ApplyDetails
+                    {
+                        UKPRN = "11112222"
+                    }
+                }
+            };
+
+            var oversightExternalComments = "oversight external comments";
+
+            _apiClient.Setup(x => x.GetOversightReview(_applicationId)).ReturnsAsync(new GetOversightReviewResponse {ModerationApproved = false, ExternalComments = oversightExternalComments});
+          
+            var result = await _service.BuildApplicationSummaryViewModelWithModerationDetails(submittedApp, _emailAddress);
+
+            result.ModerationPassOverturnedToFail = true;
```

Comment:
Is this for completeness over anything else? Doesn't look like it alters the Assertions